src/Services/CreditCards/CreditCardsDto.cs

```diff
@@ -0,0 +1,31 @@
+﻿using CloudDrive.Domain.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Services.CreditCards
+{
+    public class CreditCardsDto
+    {
+        public string HolderName { get; set; }
+        public string CreditCardNumber { get; set; }
+        public string CardSecretCode { get; set; }
+        public string ExpireMonth { get; set; }
+        public string ExpireYear { get; set; }
+
+        public CreditCardsDto ToCreditCardsDto(UserCreditCard userCreditCard)
```

Comment:
you can create a constructor that accepts a credit card instead of creating an empty object then filling it out

Reply:
ok, it is a better approach thank you